.gitlab-ci.yml

```diff
@@ -962,10 +967,13 @@ workflow:
   - test/new-e2e-utils/**/*
   - test/new-e2e/go.mod
   - tasks/security_agent.py
+  - tasks/kmt.py
+  - tasks/kernel_matrix_testing/*
 
 .on_security_agent_changes_or_manual:
   - <<: *if_main_branch
     allow_failure: true
+  - <<: *if_run_all_kmt_tests
```

Comment:
Nit: the rules order is different in `.on_security_agent_changes_or_manual` / `on_any_kmt_test_component_or_manual` (main > all KMT tests > mergequeue) and in `.on_system_probe_or_e2e_changes_or_manual` (main > mergequeue > all KMT tests).

I think both jobs should use the main > mergequeue > all KMT tests order.

Reply:
Changed, thx!